ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -0,0 +1,58 @@
+# SONiC
+
+# Setup Platform
+#- include: platform-cavm.yml
+
+# Install Cavium Host Interface Driver
+- block:
+  - name: Rmmod for driver
+    modprobe: name=xp80-Pcie-Endpoint state=absent
+
+  - name: Install driver package
+    apt: pkg=xp80PcieEndpoint
+         state=present
+         force=yes
+
+  - name: Create a list of module dependencies
+    command: depmod -a
+
+  when: sonic_hwsku != "XP-SIM"
+  become: true
+  tags: drivermod
+
+# Setup port SKU
+- name: Setup port configuration file
+  become: yes
+  template: src=etc/ssw/{{ sonic_hwsku }}/profile.ini.j2
+            dest=/etc/ssw/{{ sonic_hwsku }}/profile.ini
+            owner=root
+            group=root
+            mode=0644
+  tags: syncd
+
+- name: Setup port configuration file
+  become: yes
+  shell: ln -s -f /etc/ssw/{{ sonic_hwsku }}/port_config_{{ sonic_portsku }}.ini /etc/ssw/{{ sonic_hwsku }}/port_config.ini
```

Comment:
Who need this? Can it handle the path inside a container? Or just use original path?